addon/components/nrg-new-features.js

```diff
@@ -1,47 +1,53 @@
-import Component from '@ember/component';
+import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
-import { computed } from '@ember/object';
-import { alias, and } from '@ember/object/computed';
 import { htmlSafe } from '@ember/string';
-import layout from '../templates/components/nrg-new-features';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 
-export default Component.extend({
-  layout,
+const defaultWhatsNewText = "What's New";
+const defaultHeaderText = 'Dismiss';
```

Comment:
I think these values got mixed up:
```suggestion
const defaultHeaderText = "What's New";
const defaultButtonText = 'Dismiss';
```